qa/vagrant/src/test/resources/packaging/utils/packages.bash

```diff
@@ -117,8 +117,9 @@ verify_package_installation() {
         assert_file "/etc/default/elasticsearch" f root elasticsearch 660
 
         # Doc files
-        assert_file "/usr/share/doc/elasticsearch" d root root 755
-        assert_file "/usr/share/doc/elasticsearch/copyright" f root root 644
+        local docs=$(readlink -f /usr/share/doc/elasticsearch*)
```

Comment:
This is not really about docs since the only file that we put in this path is the `copyright` file on Debian-based systems only. I would prefer the name not be misleading about the contents of this directory.

Additionally, I think that we can use `${PACKAGE_NAME}` instead of `elasticsearch*` here as that should be set appropriately.

Reply:
Would you prefer `copyrightDir` instead of `docs`?

Reply:
Talked to Jason on another channel,  ̀copyrightDir` is okay.